controllers/indexController.js

```diff
@@ -11,26 +11,23 @@ module.exports = (connection) => {
   const controllerMethods = {};
 
   controllerMethods.render = async (req, res) => {
-    const cars = await carsModel.getAllCars(req, res);
+    const cars = await carsModel.getAllCars();
+    const allCategories = await catalogModel.getCategories();
 
     const garage = [];
 
     const options = {
       title: TITLES.MAIN,
       isHome: true,
       cars,
+      allCategories,
       garage,
     };
 
     if (req.user) {
-      const carsModificationsId =
-        await carsModel.getCarsModFromGarageWhereUserId(req.user.id);
-
-      for (const modId of carsModificationsId) {
-        const car = await carsModel.getModificationInfo(modId);
-        garage.push(car);
-      }
-      options.garage = garage.reverse();
+      const cars = await carsModel.getCarsInfoFromGarageByUser(req.user.id);
```

Comment:
The same situation with `catalogModel`, `carsModel` shouldn't know about user

Reply:
When I render index page, I see if user authorize or no.
If user authorize, I take cars from garage (DB) and I need to know his id, but if he is not authorize I take cars from local storage.